src/main/telemetry/hott.c

```diff
@@ -209,12 +211,34 @@ void hottPrepareGPSResponse(HOTT_GPS_MSG_t *hottGPSMessage)
 }
 #endif
 
+static bool time_for_Voice_Alarm(void)
+{
+	return ((alarmSound_currentTime - previous_alarmSound_time) >= (useHottAlarmSoundPeriod() * 1000000));
```

Comment:
`1000000` is a magic number.  replace with a suitable `#define`, perhaps `MICROSECONDS_IN_A_SECOND`.